app/assets/javascripts/admin/addon/routes/admin-revamp.js

```diff
@@ -4,6 +4,7 @@ import DiscourseURL from "discourse/lib/url";
 import DiscourseRoute from "discourse/routes/discourse";
 import I18n from "discourse-i18n";
 
+// DEPRECATED: (martin) This route is deprecated and will be removed in the near future.
```

Comment:
Wonder if we shall replace this "near future" with an arbitrary date in the near future?

Reply:
@Drenmi Osama is doing it quite soon, he's just deleting all this as part of his about page work + config area stuff. Noone is using it or relying on it, it was just an experiment that I made.